db/data/20220526103325_fix_general_sciences_funding.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class FixFundingMethodsFor20192020 < ActiveRecord::Migration[6.1]
+  UNDERGRAD_BURSARY =
+    OpenStruct.new(
+      training_route: TRAINING_ROUTE_ENUMS[:provider_led_undergrad],
+      amount: 9_000,
```

Comment:
Is this correct amount? The [`training_routes.rb`](http://github.com/DFE-Digital/register-trainee-teachers/blob/5b598dc5477d84bad8ef1da79b7a48e1a313f5e3/config/initializers/training_routes.rb#L132-L132) suggests it's `7000`

Reply:
You are correct, It should be 7000. Ill double check the other parameters as well